server/webapp/WEB-INF/web.xml

```diff
@@ -68,6 +68,11 @@
     <url-pattern>/*</url-pattern>
   </filter-mapping>
 
+  <filter-mapping>
```

Comment:
Had to rejig the order of filters, to avoid this url from being processed by url-rewrite filter. This filter was sending the request over to rails.